xwiki-platform-core/xwiki-platform-extension/xwiki-platform-extension-handlers/xwiki-platform-extension-handler-xar/src/main/java/org/xwiki/extension/xar/script/XarExtensionScriptService.java

```diff
@@ -72,7 +72,7 @@
     /**
      * The install request property that specifies which user triggered the XAR repair job.
      */
-    private static final String PROPERTY_USER_REFERENCE = "user.reference";
+    protected static final String PROPERTY_USER_REFERENCE = "user.reference";
```

Comment:
Note that this is introducing a new API. If it's only needed for the test, I think it could be better to have it duplicated there.

Reply:
Yes not very happy to expose something publicly just for this test but there is an easy fix: this string is now available on AbstractExtensionValidator#PROPERTY_USER_REFERENCE so IMO you should simply delete XarExtensionScriptService#PROPERTY_USER_REFERENCE and use that one instead.